package/src/index.ts

```diff
@@ -0,0 +1,288 @@
+	 * 	- Information about every kill in the game including killer, victim, assist, etc
+	 * @param {string} name - The Riot ID username of the player
+	 * @param {string} tag - The Riot ID tag of the player
+	 * @param {Region} region - The region of the player
+	 * @param {Mode} gamemodeFilter - Filter results based on gamemode
+	 * @param {ValorantMap} mapFilter - Filter results based on map
+	 * @param {any} size - Filter results based on ?? **{needs testing)**
+	*/
+	async getMatches(name: string, tag: string, region: Region, gamemodeFilter?: Mode, mapFilter?: ValorantMap, size?: any) {
+        this._validateArgs({ name, tag, region })
+        return this._fetch<MatchesResponse>(`v3/matches/${region}/${name}/${tag}`, { filter: gamemodeFilter, map: mapFilter, size })
+    }
+
+	/**
+	 * Gets the most recent 5 matches by a players PUUID
+	 * - **Returns:**
+	 * - Info about most recent 5 matches including:
+	 * 	- Metadata info about the match such as length, time, map, score, etc
+	 * 	- Information about every player including their PUUID, Riot ID, kills, ability usage, etc
+	 * 	- Information about every round in the match such as plant/defuse info, etc
+	 * 	- Information about every kill in the game including killer, victim, assist, etc
+	 * @param {string} puuid - The PUUID username of the player
+	 * @param {Region} region - The region of the player
+	 * @param {Mode} gamemodeFilter - Filter results based on gamemode
+	 * @param {ValorantMap} mapFilter - Filter results based on map
+	 * @param {any} size - Filter results based on ?? **{needs testing)**
+	*/
+	async getMatchesByPUUID(puuid: string, region: Region, gamemodeFilter?: Mode, mapFilter?: ValorantMap, size?: any) {
+        this._validateArgs({ name: puuid, region })
+        return this._fetch<MatchesResponse>(`by-puuid/matches/${region}/${puuid}`, { filter: gamemodeFilter, map: mapFilter, size })
+    }
+
+	/**
+	 * Get general information about a player from their Riot ID
+	 * - **Returns:**
+	 * - Their PUUID
+	 * - Their region
+	 * - Their account level
+	 * - Their current card
+	 * @param {string} name The Riot ID username of the player
+	 * @param {string} tag The Riot ID tag of the player
+	 * @param {boolean} force Force data update?
+	 */
+	async getProfile(name: string, tag: string, force?: boolean) {
+        this._validateArgs({ name, tag })
+        return this._fetch<ProfileResponse>(`v1/account/${name}/${tag}`, { force })
+    }
+
+	/**
+	 * Get general information about a player from their PUUID
+	 * - **Returns:**
+	 * - Their PUUID
+	 * - Their region
+	 * - Their account level
+	 * - Their current card
+	 * @param {string} puuid The PUUID of the player
+	 * @param {boolean} force Force data update?
+	*/
+	async getProfileByPUUID(puuid: string, force?: boolean) {
+        this._validateArgs({ name: puuid })
+        return this._fetch<ProfileResponse>(`v1/by-puuid/account/${puuid}`, { force })
+    }
+}
```

Comment:
There are multiple Indentation errors in this file.

Reply:
See my comment